spec/controllers/application_controller/advanced_search_spec.rb

```diff
@@ -59,3 +59,33 @@
     end
   end
 end
+
+describe VmOrTemplateController, "::AdvancedSearch" do
+  before do
+    stub_user(:features => :all)
+    controller.instance_variable_set(:@sb, {})
+  end
+
+  describe "#adv_search_redraw_left_div" do
+    before do
+      allow(controller).to receive(:adv_search_redraw_listnav_and_main)
+      controller.instance_variable_set(:@edit, :in_explorer => true)
+    end
+
+    context 'Templates & Images accordion' do
+      before do
+        controller.instance_variable_set(:@sb, :active_tree => :templates_images_filter_tree)
```

Comment:
You can drop this `before` block completely and just set the `sb` again in a `let`:
```ruby
let(:sb) { {:active_tree => :templates_images_filter_tree} }
˙˙˙